src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -339,7 +343,11 @@ internal bool VerifySetDefaults(CookieVariant variant, Uri uri, bool isLocalDoma
             }
 
             // Check the name
-            if (string.IsNullOrEmpty(m_name) || m_name[0] == '$' || m_name.IndexOfAny(ReservedToName) != -1 || m_name[0] == ' ' || m_name[m_name.Length - 1] == ' ')
+            if (string.IsNullOrEmpty(m_name)
+                || m_name.StartsWith('$')
+                || m_name.StartsWith(' ')
+                || m_name.EndsWith(' ')
+                || m_name.IndexOfAny(ReservedToName) >= 0)
```

Comment:
This library has several instances of trailing `||` and no instances of leading `||`.  Match existing style (\<smirk>especially since it's the better one\</smirk>).

```suggestion
            if (string.IsNullOrEmpty(m_name) ||
                m_name.StartsWith('$') ||
                m_name.StartsWith(' ') ||
                m_name.EndsWith(' ') ||
                m_name.IndexOfAny(ReservedToName) >= 0)
```